core/domain/rights_domain_test.py

```diff
@@ -110,70 +109,55 @@ def test_validate_voice_artist_cannot_be_viewer(self) -> None:
-        self
-    ) -> None:
-        with self.assertRaisesRegex( # type: ignore[no-untyped-call]
-            Exception, 'Cannot get activity rights for unknown activity type'):
-            rights_manager.update_activity_first_published_msec( # type: ignore[no-untyped-call]
-                'invalid_activity', 'activity_id', 0.0)
-
     def test_check_cannot_access_activity_with_no_activity_rights(self) -> None:
-        self.assertFalse(rights_manager.check_can_access_activity( # type: ignore[no-untyped-call]
+        self.assertFalse(rights_manager.check_can_access_activity(
             self.owner, None))
 
     def test_check_cannot_edit_activity_with_no_activity_rights(self) -> None:
-        self.assertFalse(rights_manager.check_can_edit_activity( # type: ignore[no-untyped-call]
+        self.assertFalse(rights_manager.check_can_edit_activity(
             self.owner, None))
 
     def test_check_cannot_voiceover_activity_with_no_activity_rights(
         self
     ) -> None:
-        self.assertFalse(rights_manager.check_can_voiceover_activity( # type: ignore[no-untyped-call]
+        self.assertFalse(rights_manager.check_can_voiceover_activity(
             self.owner, None))
 
     def test_cannot_save_activity_with_no_activity_rights(self) -> None:
-        self.assertFalse(rights_manager.check_can_save_activity( # type: ignore[no-untyped-call]
+        self.assertFalse(rights_manager.check_can_save_activity(
             self.owner, None))
 
     def test_check_cannot_delete_activity_with_no_activity_rights(self) -> None:
-        self.assertFalse(rights_manager.check_can_delete_activity( # type: ignore[no-untyped-call]
+        self.assertFalse(rights_manager.check_can_delete_activity(
             self.owner, None))
 
     def test_check_cannot_modify_activity_roles_with_no_activity_rights(
         self
     ) -> None:
-        self.assertFalse(rights_manager.check_can_modify_core_activity_roles( # type: ignore[no-untyped-call]
+        self.assertFalse(rights_manager.check_can_modify_core_activity_roles(
             self.owner, None))
 
     def test_check_cannot_release_ownership_with_no_activity_rights(
         self
     ) -> None:
-        self.assertFalse(rights_manager.check_can_release_ownership( # type: ignore[no-untyped-call]
+        self.assertFalse(rights_manager.check_can_release_ownership(
             self.owner, None))
 
     def test_check_cannnot_publish_activity_with_no_activity_rights(
         self
     ) -> None:
-        self.assertFalse(rights_manager.check_can_publish_activity( # type: ignore[no-untyped-call]
+        self.assertFalse(rights_manager.check_can_publish_activity(
             self.owner, None))
 
     def test_check_cannot_publish_activity_with_cloned_from(self) -> None:
-        self.activity_rights.cloned_from = True
-        self.assertFalse(rights_manager.check_can_publish_activity( # type: ignore[no-untyped-call]
+        self.activity_rights.cloned_from = 'True'
```

Comment:
```suggestion
        self.activity_rights.cloned_from = 'abcdefgh'
```
Keeping it `'True'` is a bit confusing as you wonder whether it should be bool or not.

Reply:
Done!